dockerfile/cuda11.1.1.dockerfile

```diff
@@ -130,6 +130,7 @@ ADD third_party third_party
 RUN make -C third_party cuda
 
 ADD . .
-RUN python3 -m pip install .[nvworker] && \
+RUN python3 -m pip install --upgrade pip && \
+    python3 -m pip install -U .[nvworker] && \
```

Comment:
why not fix the version for specific package instead? I think pandas has issue because they release a new version recently
upgrading to latest may have more issues for some other packages, like pytorch

Reply:
When I read it, I also have the same feeling on this...